tests/s25Main/mapGenerator/testRandomMap.cpp

```diff
@@ -72,7 +72,7 @@ BOOST_AUTO_TEST_CASE(GenerateRandomMap_returns_valid_water_map)
     loadGameData(worldDesc);
     MapSettings settings;
 
-    settings.size = getRandomMapSize(83, 100); // Need enough space for player islands
+    settings.size = getRandomMapSize(84, 100); // Need enough space for player islands
```

Comment:
Seems this is still not enough. Have you tried it locally with the minimum size and MAX_PLAYERS? Maybe the change here makes the player islands to small? Because previously this test worked with those sizes, didn't it?

Reply:
I think that fixed minimum island size of 200 might be too small. I'll try playing around locally.

Reply:
Or 200 is too large which would lead to a similar result (e.g. 6 players get a large island and there's not enough space for the remaining player).